### PR TITLE
Removed tip to activate builder from settings in the Editor compatibility modal

### LINKED ISSUE #10309: Change message in Editor compatibility dialog
Related to #9731 

cc @xavijam

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #10309 

screen shot 2016-10-25 at 09 57 15

CR @xavijam

### CODE REVIEW

**Comment by xavijam** [comment]
@brittanymicek for reviewing the text please. Also, I'd offer to change to the Builder, just saying something like: "If you want to enable the Builder again, just [contact us](support@cartodb.com)"

**Comment by xavijam** [comment]
cc @javisantana

**Comment by brittanymicek** [comment]
@xavijam Do you mean the Editor? ("If you want to enable the Editor, contact us.")

**Comment by xavijam** [comment]
No, the Builder. That modal will appear in the old editor.

**Comment by brittanymicek** [comment]
@xavijam 
WAIT! This map contains new Builder features that are not supported in the Editor.
If you continue to the Editor, the map may not display correctly and some features will not be available. It is recommended that you use Builder to edit and build maps.

[Remain in Builder] [Continue with Editor]

*Question: Is there a way to highlight the "Remain in Builder" button as opposed to the "Continue with Editor" button? I think that if the message above is recommending that the user remain in Builder then the emphasis should be to indicate that they stay with Builder (by highlighting the choice or making it blue).

For modal in the Editor: "To enable Builder again, please contact us."

**Comment by xavijam** [comment]
No no, if the user has that message means it has the Editor enabled, but it had the Builder enabled at some point.

**Comment by brittanymicek** [comment]
@xavijam yes, I understand! (Two separate copy issues, I just included them in one response). Does this not work?

"To enable Builder again, please contact us."

**Comment by MariaCheca** [comment]
@xavijam @brittanymicek I just updated the content, take a look at the screenshot in the PR description. Thank you both!

File: lib/assets/javascripts/cartodb/common/dialogs/builder_features_warning/template.jst.ejs [hunk 1]
```diff
@@ -7,8 +7,8 @@
   </p>
   <p class="Dialog-headerText">
     If you continue, the map may not display correctly and some features will not be available.<br />
-    It is highly recommended that you use the Builder to edit and build maps.<br />
-    <b>Tip:</b> Activate the Builder as your default tool from your Account settings.
+    It is recommended that you use Builder to edit and build maps.<br />
+    To enable Builder again, please <a href="support@carto.com">contact us</a>.
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/builder_features_warning/template.jst.ejs]: mailto please

**Pull request merged**